src/Shared/PasswordGenerator.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography;
+
+namespace Aspire;
+
+internal abstract class PasswordGenerator
+{
+    // Some chars are excluded:
+    // - prevent potential confusions, e.g., 0,o,O and i,I,l
+    // - exclude special chars which could interfere with command line arguments or connection strings, e.g. =,$,...
+
+    internal const string LowerCaseChars = "abcdefghjkmnpqrstvwxyz"; // exclude i,l,u,o
+    internal const string UpperCaseChars = "ABCDEFGHJKMNPQRSTVWXYZ"; // exclude I,L,U,O
```

Comment:
Why are these characters being excluded?

Reply:
It can lead to unfriendly words.